gear-programs/vft-treasury/src/wasm/tests/vft_treasury.rs

```diff
@@ -0,0 +1,159 @@
+use gtest::{Program, System};
+use sails_rs::prelude::*;
+use utils::{VftTreasury, *};
+use vft_treasury_app::services::error::Error;
+
+mod utils;
+
+fn setup_for_test(system: &System) -> (Program<'_>, Program<'_>, Program<'_>) {
```

Comment:
It's not clear in which order programs are returned. Consider introducing new `struct` for it